test/e2e/util_gatewayapi_test.go

```diff
@@ -94,7 +67,7 @@ func assertCrdExists(t *testing.T, crdname string) (string, error) {
 		crdConditions := crd.Status.Conditions
 		for _, version := range crd.Spec.Versions {
 			if version.Served {
```

Comment:
How about this?  We don't really care about storage versions, right?
```suggestion
			if version.Served  && !version.Storage {
```

Reply:
I'm not sure why we care about the served versions at all in the test output.  However, if it ever became necessary to know, then it seems like it would be more useful to have the full list of served versions.  If anything, it might be useful _also_ to know which one were the stored version.  In any case, I don't want to put in a lot of extra logic in an attempt to anticipate and log exactly the information that could be useful in some hypothetical situation.  (I'm tempted to simplify the logic to simply `t.Logf("%+v", crd)`.)

Reply:
I don't think we consider either served or stored when we look at CRDs.  Would that pose a problem?

Reply:
I'm not sure what you mean.  Do you mean when the test looks at the CRDs, it doesn't verify that the expected version is served/stored?